README.md

```diff
@@ -53,6 +53,7 @@ as ``ovirt_engine_setup_answer_file_path``.
 | ovirt_engine_setup_fqdn               | UNDEF                 | Host fully qualified DNS name of the server. |
 | ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
 | ovirt_engine_setup_firewall_manager   | firewalld             | Specify the type of firewall manager to configure on Engine host, following values are availableL: `firewalld`,`iptables` or empty value to skip firewall configuration. |
+| ovirt_engine_setup_apache_ssl		| UNDEF			| By default, we omit this variable. Engine default of `automatic` then kicks in, using engine CA to sign HTTP certificates. When set to `manual`, you'll have to take care of all the TLS settings yourself, here are [RHV 4.3 docs](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html-single/administration_guide/index#Replacing_the_Manager_CA_Certificate). |
```

Comment:
I wouldn't use downstream documentation link in upstream code, can you please use this instead:
https://www.ovirt.org/documentation/admin-guide/appe-oVirt_and_SSL.html